src/jobflow_remote/config/base.py

```
@@ -177,6 +177,12 @@ class WorkerBase(BaseModel):
         None,
         description="Options for batch execution. If define the worker will be considered a batch worker",
     )
+    scheduler_username: Optional[str] = Field(
+        None,
+        description="If defined, the list of jobs running on the worker will be fetched based on the"
+        "username instead that from the list of job ids. May be necessary for some "
+        "scheduler_type (e.g. SGE)",
+    )
```

Just one question here. For sge, it is mandatory to use this option, otherwise it won't work, should there be something to prevent *not* having it defined (or maybe jf project check can fail if it's not set ?) ?

---

It would be relatively easy to add this check. But apparently it is then possible to query by id. I am not sure if this should be a strict limitation.